manifests/capture.archons.yaml

```
@@ -0,0 +1,16 @@
+﻿name: capture.archons
+authors: Nova.
+homepage: https://github.com/Nova1422/archons
+license: ALL-RIGHTS-RESERVED
+version: v1.0
+shortDescription: Spawns captureable Archon ships
+description: >
+  Spawns captureable Archon ships in the system "Persitar" every ten minutes.
+  PS - Pre-existing Archons can't be captured nor will they be angered.
+  Capturing the spawned Archons will not be any easier.
+url: https://github.com/Nova1422/archons/archive/refs/tags/$version.zip
+iconUrl: https://github.com/Nova1422/archons/blob/main/icon.png
```

These need to be versioned, i.e. belonging to the v1.0 release. Easiest way to do that is to take the urls from the autoupdate-block below, and replace the `$version`:


```suggestion
url: https://github.com/Nova1422/archons/archive/refs/tags/v1.0.zip
iconUrl: https://raw.githubusercontent.com/Nova1422/archons/v1.0/icon.png
```